scripts/cnv_wdl/germline/cnv_germline_case_workflow.wdl

```diff
@@ -161,7 +162,7 @@ workflow CNVGermlineCaseWorkflow {
 
     call CNVTasks.ScatterIntervals {
         input:
-            interval_list = PreprocessIntervals.preprocessed_intervals,
+            interval_list = filtered_intervals,
```

Comment:
> Just to make sure I have this right -- in the case workflow you're collecting read counts over the full set of intervals (intervals), but then you're running the caller over the filtered intervals? Is that because you want the full data for contig ploidy determination?

Coverage collection is performed over all intervals, but ploidy determination and gCNV are run over the filtered intervals.  So changing the filtering parameters effectively masks the data for model fitting.

> It looks like the task GermlineCNVCallerCaseMode gets intervals that it never uses. Ditto annotated_intervals. Is the scatter accomplished by only passing one shard of the model?

Oops, good catch.  At some point, I think we switched things so that the intervals for each shard are contained in the corresponding model tar, rather than passing them through  `-L`.  I think I removed some vestigial documentation elsewhere in this PR, but forgot to clean up the WDL.

Technically, I guess we don't have to run ScatterIntervals in case mode, since we could just scatter over the model tars.  However, this step is cheap, and I think that having the scatter parameters explicit in the case mode JSON is not too much hassle.  The user just has to make sure that they use the same parameters for cohort and case mode.  Same logic goes for the PreprocessIntervals step.  However, we can always change this later so that all necessary results produced in cohort mode are simply taken as input to case mode, if desired.  There are pros and cons to both.